scripts/temp.deployment.sh

```diff
@@ -21,24 +22,50 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-# Temporary script to deploy argo onto a system for testing
-#     NOTE: this will be removed once we have deployment work done
+set -e
 
-# csm 1.2
+function deployNLS() {
+    BUILDDIR="/tmp/build"
+    mkdir -p "$BUILDDIR"
+    kubectl get secrets -n loftsman site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d > "${BUILDDIR}/customizations.yaml"
+    kubectl get configmap -n loftsman loftsman-platform -o jsonpath='{.data.manifest\.yaml}' > "${BUILDDIR}/iuf.yaml"
+    manifestgen -i "${BUILDDIR}/iuf.yaml" -c "${BUILDDIR}/customizations.yaml" -o "${BUILDDIR}/platform.yaml"
+    yq w -i "${BUILDDIR}/platform.yaml" 'spec.charts[0].version' "$2"
+    charts="$(yq r /tmp/build/platform.yaml 'spec.charts[*].name')"
+    for chart in $charts; do
+        if [[ $chart != "cray-iuf" ]] && [[ $chart != "cray-nls" ]]; then
```

Comment:
Does this only work on a system that has already been installed with a recent enough version of CSM to have the `cray-iuf` chart in the `platform.yaml` loftsman manifest? Is that a problem? I.e., do we need to deploy/update IUF on systems with earlier versions of CSM?